package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentstack/datasync-content-store-filesystem",
-  "version": "2.0.6",
+  "version": "2.0.7",
```

Comment:
@vkalta , pls do minor version bump as it's enhancement.

Reply:
Done @aman19K

Reply:
Done @aman19K